scripts/obtain/dotnet-install.sh

```diff
@@ -84,6 +84,10 @@ get_current_os_name() {
                     echo "opensuse.13.2"
                     return 0
                     ;;
+                "opensuse.42.1")
```

Comment:
I was confused to see this file checked in here at all, looks like it got added at some point https://github.com/dotnet/core-setup/commit/5caaef7647fee38da701f3772aa3e710ddd10ccc

This is core-setup's special copy of this file and it looks to be only checked in for unix (not windows), my comment was just that in order for everyone else to get these changes they'll need to go into cli as well.